std_msgs/Header header
float64 cam_survey_time             # cam_survey_time reading
float64 cam_survey_time_var         # cam_survey_time_var reading

